lib/gps.js

```diff
@@ -189,7 +189,7 @@ function GPS(opts) {
 
   // If necessary set default property values
   this.fixed = opts.fixed || 6;
-  this.baud = opts.baud || breakout.baud || chip.baud;
+  this.baud = opts.baud || this.defaultBaud;
```

Comment:
Instead of making a new property, just do: 

```js
  DEFAULT: {
    baud: {
      writable: true,
      value: 9600
    },
```
and

```js
  MT3339: {
    baud: {
      writable: true,
      value: 9600
    },
```